app.js

```diff
@@ -63,4 +63,274 @@ function fillData() {
   textArea.value = desirializedObject.textArea;
 }
 
-window.addEventListener('load', fillData);
\ No newline at end of file
+window.addEventListener('load', fillData);
+
+// create the card information
+const otherProjects = [
+  {
+    name: 'Multi-Post Stories',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    image: './assets/first-card.png',
+    desktopImg: './assets/large.png',
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    name: 'Profesional Art Printing Data',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    image: './assets/rest-card.png',
+    desktopImg: './assets/Img-Placeholder.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
```

Comment:
- [ ] The same issue affects the desktop version. Part of the modal is cut off at the bottom of the popup. Kindly look into it and fix it.

### Your Design
![desktop-modal](https://user-images.githubusercontent.com/26597432/202853274-c754f021-ce15-4224-81d1-36f60d62ad1f.png)